src/Data/Migrations/20210708111637_AlterDetectionsAddFields.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Data.Migrations
+{
+    using System;
+    using Microsoft.EntityFrameworkCore.Migrations;
+
+    public partial class AlterDetectionsAddFields : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "Class",
+                table: "Detections",
+                type: "text",
+                nullable: true);
```

Comment:
Let's make sure that this column is not null able, as no `EDetection` should be created without the `class` property being set.

Reply:
Read [this](https://docs.microsoft.com/en-us/ef/core/modeling/entity-properties?tabs=data-annotations%2Cwithout-nrt) article on data annotations to figure out how to do it.